include/ignition/gui/qml/PlottingInterface.qml

```diff
@@ -0,0 +1,556 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+import QtQuick 2.9
+import QtCharts 2.2
+import QtQuick.Controls 2.2
+import QtQuick.Controls.Styles 1.4
+import QtQuick.Controls.Material 2.1
+import QtQuick.Layouts 1.3
+import Qt.labs.platform 1.0
+import "qrc:/qml"
+
+Rectangle
+{
+  id : main
+
+  Layout.minimumWidth: 600
+  Layout.minimumHeight: 600
+  anchors.fill: parent
+  color: (Material.theme == Material.Light) ? Material.color(Material.Grey,Material.Shade100) : "transparent"
+
+  property var charts: ({})
+  property int mainChartID: 1
+
+  property int idIncrementor: 0
+
+  property bool multiChartsMode: false
+
+  // Horizonal Layout to hold multi charts (small charts)
+  Rectangle {
+      id: rowCharts
```

Comment:
Could you change the indentation of this and all QML files to be 2 spaces consistently?